python/ray/_private/ray_logging/filters.py

```diff
@@ -20,4 +20,13 @@ def filter(self, record):
             task_id = runtime_context.get_task_id()
             if task_id is not None:
                 setattr(record, LogKey.TASK_ID.value, task_id)
+            task_name = runtime_context.get_task_name()
+            if task_name is not None:
+                setattr(record, LogKey.TASK_NAME.value, task_name)
+            task_function = runtime_context.get_task_function()
```

Comment:
`get_task_function_name`

`get_task_function` makes me feel it might return the FunctionDescriptor